ingestion/src/metadata/ingestion/source/database/bigquery/queries.py

```diff
@@ -81,7 +82,7 @@
     user_email as user_name
   FROM `region-{region}`.INFORMATION_SCHEMA.JOBS_BY_PROJECT
   WHERE statement_type = 'SCRIPT'
-    AND start_time >= '{start_date}'
+    AND creation_time >= '{start_date}'
```

Comment:
is this change transparent? Do we get the same results when fetching for stored procedures?

Reply:
`start_time field represents the time when the job transitions from the PENDING state to either RUNNING or DONE`

so IMO there should not be a huge difference between the creation_time and start_time and also this means that start_time would always be > creation_time so it would not make sense it we apply filter on both the start_time

Reply:
sounds good, thanks. Let's keep it as this